pages/api/teams/[slug]/index.ts

```diff
@@ -57,10 +59,16 @@ const handlePUT = async (req: NextApiRequest, res: NextApiResponse) => {
   const teamMember = await throwIfNoTeamAccess(req, res);
   throwIfNotAllowed(teamMember, 'team', 'update');
 
+  const { name, slug, domain } = req.body;
+
+  if (!validateDomain(domain)) {
```

Comment:
@Dhoni77 `domain` should be optional. That means validate it only if it is not empty.

Reply:
done